backend/src/graphql/validator/ContributionStatusArray.ts

```diff
@@ -0,0 +1,21 @@
+import { registerDecorator, ValidationOptions } from 'class-validator'
+
+import { ContributionStatus } from '@enum/ContributionStatus'
+
+export function isContributionStatusArray(validationOptions?: ValidationOptions) {
```

Comment:
It seems to be on expert-level, i can't follow this construction:
- which return type does the isContributionStatusArray() have?
- where is the invocation of the internal function registerDecorator()?
- how can the intenal function return object and string without defining a return type in the outer function?
- ...